tests/client/amo/components/TestSearchResult.js

```diff
@@ -53,7 +53,14 @@ describe('<SearchResult />', () => {
 
   it('renders the user count', () => {
     const users = findRenderedDOMComponentWithClass(root, 'SearchResult-users');
-    assert.equal(users.textContent, '553 users');
+    assert.equal(users.textContent, '5,253 users');
+  });
+
+  it('localises the user count', () => {
+    const localisedRoot = renderResult(result, { lang: 'fr' });
+    const users = findRenderedDOMComponentWithClass(localisedRoot, 'SearchResult-users');
+    // \xa0 is a non-breaking space.
+    assert.match(users.textContent, /5\xa0253/);
```

Comment:
L'espace est bizarre ici pour moi.

How many results?
"5 253 !"

English must feel so cramped in comparison. 😉